web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -539,6 +544,11 @@ import {get_string_diff} from "../../src/util";
   }
 
   , blur: function (e) {
+      // Blurs that move focus to elsewhere within the parent element shouldn't
+      // hide the typeahead.
+      if ($(e.relatedTarget).parents(this.options.parentElement).length > 0) {
+        return;
+      }
```

Comment:
Can you provide an example of this and an argument for this change won't potentially make weird things happen with other typeahead components like the compose box? I'm a bit confused as to what this does and unlike the other typeahead changes, it's not optional.

Reply:
I can copy the example from above here, or would you want something more?

> *   This allows us to have things like a close button, and be able
> *   to move focus there without the typeahead closing.

Essentially I want to allow focus to move within the input field without the typeahead closing. I don't think this is relevant to anything else becaues `parentElement` is only used for search.

Reply:
Ohh, OK, it's an option we added that we only pass there.